webapp/channels/src/components/login/login.test.tsx

```diff
@@ -318,10 +318,9 @@ describe('components/login/Login', () => {
 
         const button = screen.getByRole('link', {name: 'Gitlab Icon GitLab 2'});
 
-        expect(button.style).toMatchObject({
-            color: 'rgb(0, 255, 0)',
-            borderColor: '#00ff00',
-        });
+        // In jsdom 25+, style is a CSSStyleDeclaration and colors are normalized to rgb
+        expect(button.style.color).toBe('rgb(0, 255, 0)');
+        expect(button.style.borderColor).toBe('rgb(0, 255, 0)');
```

Comment:
Due to Jest upgrade (with jsdom), style is a CSSStyleDeclaration and colors are normalized to rgb.